sites/main-site/src/content/blog/2025/refurbushing-the-pipeline-download.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Why are my pipeline download tests failing?"
+subtitle: "Refurbishing the pipeline downloads command"
+headerImage: "https://images.unsplash.com/photo-1636819488537-a9b1ffb315ce"
+headerImageAlt: "3D rendering of a cute little space rocket painted red and white with a fluffy smoke cloud coming out of it."
+pubDate: 2025-08-15T12:00:00+01:00
+authors:
+    - "ErikDanielsson"
+label:
+    - "tools"
+maxHeadingDepth: 3
+---
+
+import { Image } from "astro:assets";
+
+import old_nf_version from "@assets/images/blog/pipeline-download-refactor/old-nf-version.png";
+import rnaseq_in_action from "@assets/images/blog/pipeline-download-refactor/rnaseq-in-action.png";
+
+The `nf-core pipelines download` command, used when you want to run an `nf-core` pipeline in an offline compute environment, has recently undergone a [substantial refactor](https://github.com/nf-core/tools/pull/3634) that has recently been merged into the development version of `nf-core/tools`.
+The command has for a long time been reliant on complex regex logic to capture container strings in modules and config files.
+Due to the variability in where a pipeline can define containers, this has been prone to breaking.
+
+Not anymore!
+As of [Nextflow 25.04](https://www.nextflow.io/docs/latest/migrations/25-04.html) the `nextflow inspect` command has been substantially extended to allow fetching of all pipeline containers by executing a dry-run of the pipeline.
+This has now been integrated into the `nf-core/tools` code base which means that the download command can offload the tricky problem of finding pipeline containers to Nextflow, making the command simpler and more reliant at the same time.
+
+:::warning{title="Why are is the pipeline download tests failing?!"}
```

Comment:
```suggestion
:::warning{title="Why are the pipeline download tests failing?!"}
```

Reply:
I think explaining the new command should be explained before this section, and this can be the last one before `further details`